packages/catala-lsp/catala-lsp.0.0.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "Vincent Botbol"
+authors: [ "Vincent Botbol" ]
+license: "Apache-2.0"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml" {>= "4.14.1"}
+  "dune" { >= "3.0" }
+  "catala"    { = "dev" }
+  "logs"
+  "uri"
+  "linol"     { = "dev" }
+  "linol-lwt" { = "dev" }
+]
+pin-depends: [
+  ["catala.dev" "git+https://github.com/CatalaLang/catala#lsp-compat"]
+  # TODO: remove linol's pinning once the required changes are
+  # published in opam
+  ["linol.dev" "git+https://github.com/c-cube/linol#a779942f9592a762d26484d03b1f93110dbaf577"]
+  ["linol-lwt.dev" "git+https://github.com/c-cube/linol#a779942f9592a762d26484d03b1f93110dbaf577"]
+]
```

Comment:
The CI is flagging this. Essentially the `pin-depends` fields is for internal use, development, etc. But not for inclusion in the opam-repository. (Some exceptions can be found in the opam-repository, but they should be removed.)

How far is it for `linol` to be published in opam?  
How far is the `lsp-compat` branch from being merged and released?

Note that currently the dependency to `"dev"` means that the CI essentially doesn't run ("catala-lsp.0.0.1 (failed: [SKIP] Package not available").

Reply:
Yes, I was somewhat wary of this.

Regarding `linol`, I don't have a clue on when they are planning to do a release.
And, for `catala`, we have control over it but it's really not a priority to do a release as we have an upcoming deadline.

The initial goal was to have non-OCaml developers install the tools without much friction. I guess that for the time being it would also be reasonable to make them locally clone the repo while everything progressively lands..

Reply:
An alternative is to have them run `opam remote add …` adding your copy of the opam-repository with your added packages. It's one more step than just `opam install …` for the users.